contribs/gmf/externs/gmf-themes.js

```diff
@@ -375,6 +375,29 @@ gmfThemes.GmfFunctionalities = function() {};
  */
 gmfThemes.GmfFunctionalities.prototype.default_basemap;
 
+
+/**
+ * Whether to open the filter tool panel upon loading an application.
+ *
+ * Note: altough this is a list, it should have been boolean. The value
+ * required to have this set is actually: `['true']`.
+ *
+ * @type {Array.<!string>|undefined}
+ */
+gmfThemes.GmfFunctionalities.prototype.open_layer_filter_panel;
```

Comment:
The functionality should be `open_panel` and contain `['layer_filter']` to open that panel.